src/main/kotlin/lotto/domain/LotteryPrizeAmount.kt

```diff
@@ -0,0 +1,18 @@
+package lotto.domain
+
+enum class LotteryPrizeAmount(val winNum: Int, val prize: Int) {
+    ZERO_WIN(0, 0),
+    ONE_WIN(1, 0),
+    TWO_WIN(2, 0),
+    THREE_WIN(3, 5000),
+    FOUR_WIN(4, 50000),
+    FIVE_WIN(5, 1500000),
+    SIX_WIN(6, 2000000000),
```

Comment:
```suggestion
    THREE_WIN(3, 5_000),
    FOUR_WIN(4, 50_000),
    FIVE_WIN(5, 1_500_000),
    SIX_WIN(6, 2_000_000_000),
```
언더스코어를 활용하여 가독성을 높일수 있습니다 👍